docs/src/understand/sft.rst

```diff
@@ -26,41 +30,41 @@ without compromising end-to-end security.
 Architecture
 ------------
 
-The following diagram is centered around SFT and its role within a calling setup. Restund is seen
+The following diagram is centered around `SFT` and its role within a calling setup. Restund is seen
```

Comment:
Consider adding ports used in this section

Default SFT Port: 8585/tcp (HTTP)
SFT Metrics Port: 49090/tcp (HTTP)